include/net/socket.h

```diff
@@ -145,6 +145,18 @@ struct zsock_pollfd {
  */
```

Comment:
typo in commit msg of `net: socket: Add option to create native TLS sock with offloaded TCP`
`s/implmenetation/implementation/`